rootfs-builder/clearlinux/config.sh

```diff
@@ -3,6 +3,20 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+# Clear Linux OS identifier (name)
+OS_NAME="Clear"
+
+#Fedora version to use
+OS_VERSION=${OS_VERSION:-latest}
+
+# Fedora base repository metalink URL
+REPO_URL="https://download.clearlinux.org/current/x86_64/os/"
```

Comment:
I wonder if we need a check somewhere (`lib.sh`?) to error if `$ARCH != x86_64` for Clear Linux?